traffic_monitor/tmclient/tmclient.go

```diff
@@ -23,6 +23,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/apache/trafficcontrol/lib/go-log"
```

Comment:
Another *apache/trafficcontrol* that should be grouped separately from standard library imports.